src/binder/clrprivbinderassemblyloadcontext.cpp

```diff
@@ -251,9 +271,51 @@ HRESULT CLRPrivBinderAssemblyLoadContext::SetupContext(DWORD      dwAppDomainId,
     return hr;
 }
 
+/* static */
+void CLRPrivBinderAssemblyLoadContext::DestroyContext(CLRPrivBinderAssemblyLoadContext *pBindContext, INT_PTR ptrManagedStrongAssemblyLoadContext)
+{
+    CONTRACTL
+    {
+        GC_NOTRIGGER;
+        THROWS;
+        MODE_PREEMPTIVE;
+        SO_TOLERANT;
+    }
+    CONTRACTL_END;
+
+    GCX_COOP();
```

Comment:
ok, so I will move this to the callsite is `AssemblyNative::DestroyAssemblyLoadContext` and is using QCALL which is `MODE_PREEMPTIVE` and change this method to MODE_COOP